templates/home/index.html

```diff
@@ -135,130 +135,47 @@ <h1>Our Initiatives</h1>
             <div class="glide">
                   <div class="glide__track" data-glide-el="track">
                     <ul class="glide__slides">
+                        
+                        {% for initiative in initiatives %}
+
                           <li class="glide__slide">
                               <div class="card">
                                 <div class="card-img">
-                                    <img src='{% static "home/images/02.jpg" %}'>
+                                    <img src={{initiative.thumbnail}}>
```

Comment:
Use space after and before the curly braces. Also, enclose it in double quotes. Ex. `src="{{ initiative.thumbnail }}"`.